frontend/src/components/sidebar/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import sidebarStyles from './sidebar.module.scss';
+
+const Sidebar = () => {
+  const buttonList = [
+    { name: 'Summary' },
+    { name: 'Rota' },
+    { name: 'Employees' },
```

Comment:
objects so that later they can be given Ids to tell the app what the user has clicked and it will be nice (i feel) if ids had no spaces whereas menu options might

Reply:
I am not entirely sure what you mean by this? Also a question could the button styles be determined by the route from the router? That means we have one source of truth.

Reply:
so when the buttons are clicked one of two things will happen
1. they each have their own callback (unlikely) so we need an object to call it with
2. they have one given callback function that they call with different values (lets call them IDs), I wanted to separate ID and name because
  a. I feel IDs (entirely internal shoud be camelcase no spaces
  b. separate the display from implementation. 

Im not sure what you mean either haha, but to share button styles I would suggest exporting a @mixin from whereever we define constant colours